server/utils/utils.test.ts

```diff
@@ -45,3 +51,45 @@ describe('retrieveQuestionResponseFromSession', () => {
     expect(questionResponse).toBe('no')
   })
 })
+
+describe('removeBlankSummaryListItems', () => {
+  it('removes all blank and undefined items', () => {
+    const items = [
+      {
+        key: {
+          text: 'Names',
+        },
+        value: {
+          text: 'Name',
+        },
+      },
+      {
+        key: {
+          text: 'CRN',
+        },
+        value: {
+          text: '',
+        },
+      },
+      {
+        key: {
+          text: 'Date of Birth',
+        },
+        value: {
+          text: undefined,
+        },
+      },
+    ] as Array<SummaryListItem>
+
+    expect(removeBlankSummaryListItems(items)).toEqual([
+      {
+        key: {
+          text: 'Names',
+        },
+        value: {
+          text: 'Name',
+        },
+      },
```

Comment:
nice utility! should we also have some `html:` values to describe how those values (like the `text:` values) are either retained or discarded depending on whether they are present?

Reply:
Oh, good call 👍